Makefile

```diff
@@ -8,16 +8,20 @@ $(PROJ).pdf: lkmpg.tex
 	pdflatex -shell-escape $< 2>/dev/null >/dev/null
 
 html: lkmpg.tex html.cfg
-	make4ht --shell-escape --utf8 --format html5 --config html.cfg --output-dir html lkmpg.tex
-	ln -sf lkmpg.html html/index.html
-	rm -f  lkmpg.xref lkmpg.tmp lkmpg.html lkmpg.css lkmpg.4ct lkmpg.4tc lkmpg.dvi lkmpg.lg lkmpg.idv lkmpg*.svg lkmpg.log lkmpg.aux 
+	sed $ 's/\t/    /g' lkmpg.tex > lkmpg-for-ht.tex
+	make4ht --shell-escape --utf8 --format html5 --config html.cfg --output-dir html  lkmpg-for-ht.tex
+	ln -sf lkmpg-for-ht.html html/index.html
+	rm -f  lkmpg-for-ht.tex lkmpg-for-ht.xref lkmpg-for-ht.tmp lkmpg-for-ht.html lkmpg-for-ht.css lkmpg-for-ht.4ct lkmpg-for-ht.4tc lkmpg-for-ht.dvi lkmpg-for-ht.lg lkmpg-for-ht.idv lkmpg*.svg lkmpg-for-ht.log lkmpg-for-ht.aux
+	rm -rf _minted-$(PROJ) _minted-lkmpg-for-ht
+
+indent:
+	(cd examples; find . -name '*.[ch]' | xargs clang-format -i)
 
 indent:
```

Comment:
The `indent` target is duplicated.

Reply:
Sorry for my stupid mistakes because I'm called for dealing another task...